web/src/components/layout/Flex.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { Flex as PFFlex, FlexProps, FlexItem, FlexItemProps } from "@patternfly/react-core";
+
+/**
+ * NOTE: below code for dealing with PF/Flex types and extract the "responsive props" is a bit
+ * complex but useful for building a wrapper around such a component without the risk of getting it
+ * silently broken if PF/Flex changes these props by deleting them or adding new ones.
+ *
+ * For sure, would be better to add these responsive props shortcuts direclty in PF/Flex to allow
+ * the consumer to just set the `default` value when not needed to change it depending on
+ * the breakpoint. But at this moment we're a bit short of time for creating and testing such
+ * an elaborated PR against upstream.
+ *
+ * BTW, the lines for extracting an object from the type were borrowed from
+ * https://dev.to/scooperdev/generate-array-of-all-an-interfaces-keys-with-typescript-4hbf
+ */
+
+// NOTE: PF/Flex#order prop is missing "sm" breakpoint
+// NOTE: Don't know why these ommited props are being captured otherwise
```

Comment:
I would say that you already know that, right? (considered as *any*)

Reply:
Right!